addons/Hierarchical_Lesson_Report/src/presenter.js

```diff
@@ -210,88 +434,154 @@ function AddonHierarchical_Lesson_Report_create() {
         }
 
         if (isScoreEnable) {
-
-            if (presenter.configuration.showResults) {
-                createProgressCell(row, score, index, isChapter);
-            }
+            if (configuration.showResults) createProgressCell(row, score, index, isChapter, isInPrintableMode);
 
             if (!isChapter) {
-                presenter.lessonScore.pageCount++;
-            }
-            presenter.lessonScore.checks += score.checkCount;
-            presenter.lessonScore.mistakes += score.mistakeCount;
-            presenter.lessonScore.errors += score.errorCount;
-            presenter.lessonScore.score += pageScore;
-            presenter.lessonScore.maxScore += score.maxScore;
-            presenter.lessonScore.scaledScore += presenter.getPageScaledScore(score.maxScore, pageScore, isChapter, pageId);
-
-            if (presenter.configuration.showChecks) {
-                var checksCell = document.createElement('td');
-                $(checksCell).appendTo($(row))
-                    .addClass("hier_report-checks")
-                    .html(score.checkCount);
+                lessonScore.pageCount++;
             }
+            lessonScore.checks += score.checkCount;
+            lessonScore.mistakes += score.mistakeCount;
+            lessonScore.errors += score.errorCount;
+            lessonScore.score += pageScore;
+            lessonScore.maxScore += score.maxScore;
+            lessonScore.scaledScore += presenter.getPageScaledScore(score.maxScore, pageScore, isChapter, pageId);
+
+            if (configuration.showChecks)
```

Comment:
To również bardzo podobne do poprzednich miejsc o których wspominałem.

Reply:
Temat został omówiony na spotkaniu